platform/podjs/rollup-plugin-gen-podjs/genPodjs.js

```diff
@@ -31,7 +31,7 @@ function copyPodJs(dest) {
  */
 function loadManifest(options = {}) {
     if (!options.build_dir || !options.manifestPath) {
-        throw new Error("manifestPath or build_dir not specified");
+        throw new Error("build_dir not specified");
```

Comment:
Why eliminate manifestPath? Anyway, maybe you can make it a bit more helpful by specifiying *where* it's not specified.

Reply:
I brought that back - genPodjs didn't support features without manifests, and I resisted adding one. But the home screen doesn't work with features without manifests anymore either, so...

Reply:
Oh, thought this was outdated - it wasn't, I had actually forgotten to change that back! Done.